datajunction-server/datajunction_server/database/availabilitystate.py

```diff
@@ -70,7 +70,9 @@ def is_available(
         """
         Determine whether an availability state is useable given criteria
         """
-        # Criteria to determine if an availability state should be used needs to be added
+        # If we are materializing, we should not use this availability state
```

Comment:
Is this build criteria passed down all the way to the other nodes that this node depends on? Or is it only used for this node?

Reply:
Good question. I though it was used by all the nodes, but I couldn't see that in my tests. Need to investigate more...

Reply:
Adjusted with fresh code.